server/src/database/migrations/20230809143432_add_views_table/migration.sql

```diff
@@ -0,0 +1,36 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[fieldName,objectName,viewId]` on the table `viewFields` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- AlterTable
+ALTER TABLE "viewFields" ADD COLUMN     "viewId" TEXT;
+
+-- CreateTable
+CREATE TABLE "views" (
+    "id" TEXT NOT NULL,
+    "name" TEXT NOT NULL,
+    "objectName" TEXT NOT NULL,
```

Comment:
I think we should actually introduce:
- viewName
- objectName => objectId to prepare for customObjects: objectId=enum(person|company|object-323)
- type => table | pipeline

Reply:
Should `ViewField`'s `objectName` be renamed to `objectId` as well?

Reply:
I added `type` (+ `ViewType` enum) and renamed `objectName` to `objectId`. `name` already exists for the `View`'s name.